deps/currency-data.jl

```diff
@@ -1,158 +1,313 @@
+    :GEL => (2,  981, "Lari"),
+    :CZK => (2,  203, "Czech Koruna"),
+    :MOP => (2,  446, "Pataca"),
+    :PLN => (2,  985, "Zloty"),
+    :SSP => (2,  728, "South Sudanese Pound"),
+    :AFN => (2,  971, "Afghani"),
+    :SYP => (2,  760, "Syrian Pound"),
+    :STD => (2,  678, "Dobra"),
+    :USD => (2,  840, "US Dollar"),
+    :BND => (2,   96, "Brunei Dollar"),
+    :CVE => (2,  132, "Cabo Verde Escudo"),
+    :ARS => (2,   32, "Argentine Peso"),
+    :CDF => (2,  976, "Congolese Franc"),
+    :PGK => (2,  598, "Kina"),
+    :RUB => (2,  643, "Russian Ruble"),
+    :MNT => (2,  496, "Tugrik"),
+    :RWF => (0,  646, "Rwanda Franc"),
+    :SZL => (2,  748, "Lilangeni"),
+    :GBP => (2,  826, "Pound Sterling"),
+    :WST => (2,  882, "Tala"),
+    :BYN => (2,  933, "Belarusian Ruble"),
+    :BGN => (2,  975, "Bulgarian Lev"),
+    :JOD => (3,  400, "Jordanian Dinar"),
+    :YER => (2,  886, "Yemeni Rial"),
+    :ZWL => (2,  932, "Zimbabwe Dollar"),
+    :LYD => (3,  434, "Libyan Dinar"),
+    :TMT => (2,  934, "Turkmenistan New Manat"),
+    :FJD => (2,  242, "Fiji Dollar"),
+    :KHR => (2,  116, "Riel"),
+    :BMD => (2,   60, "Bermudian Dollar"),
+    :KZT => (2,  398, "Tenge"),
+    :XCD => (2,  951, "East Caribbean Dollar"),
+    :NPR => (2,  524, "Nepalese Rupee"),
+    :OMR => (3,  512, "Rial Omani"),
+    :CLP => (0,  152, "Chilean Peso"),
+    :ERN => (2,  232, "Nakfa"),
+    :TOP => (2,  776, "Pa’anga"),
+    :CUP => (2,  192, "Cuban Peso"),
+    :LBP => (2,  422, "Lebanese Pound"),
+    :SCR => (2,  690, "Seychelles Rupee"),
+    :NOK => (2,  578, "Norwegian Krone"),
+    :RON => (2,  946, "Romanian Leu"),
+    :MUR => (2,  480, "Mauritius Rupee"),
+    :BDT => (2,   50, "Taka"),
+    :MGA => (2,  969, "Malagasy Ariary"),
+    :SBD => (2,   90, "Solomon Islands Dollar"),
+    :ISK => (0,  352, "Iceland Krona"),
+    :GIP => (2,  292, "Gibraltar Pound"),
+    :KES => (2,  404, "Kenyan Shilling"),
+    :BWP => (2,   72, "Pula"),
+    :BBD => (2,   52, "Barbados Dollar"),
+    :KMF => (0,  174, "Comorian Franc"),
+    :XAF => (0,  950, "CFA Franc BEAC"),
+    :AMD => (2,   51, "Armenian Dram"),
+    :CNY => (2,  156, "Yuan Renminbi"),
+    :KRW => (0,  410, "Won"),
+    :SEK => (2,  752, "Swedish Krona"),
+    :BRL => (2,  986, "Brazilian Real"),
+    :CRC => (2,  188, "Costa Rican Colon"),
+    :SVC => (2,  222, "El Salvador Colon"),
+    :IRR => (2,  364, "Iranian Rial"),
+    :NZD => (2,  554, "New Zealand Dollar"),
+    :SRD => (2,  968, "Surinam Dollar"),
+    :NIO => (2,  558, "Cordoba Oro"),
+    :MKD => (2,  807, "Denar"),
+    :KGS => (2,  417, "Som"),
+    :DJF => (0,  262, "Djibouti Franc"),
+    :UGX => (0,  800, "Uganda Shilling"),
+    :VUV => (0,  548, "Vatu"),
+    :LKR => (2,  144, "Sri Lanka Rupee"),
+    :RSD => (2,  941, "Serbian Dinar"),
+    :ILS => (2,  376, "New Israeli Sheqel"),
+    :ANG => (2,  532, "Netherlands Antillean Guilder"),
+    :LSL => (2,  426, "Loti"),
 )
 
+Currency(:KYD)
```

Comment:
We can't have instances in the structure because we need the structure to check the validity of instances :)

Good idea to put the type in the structure though 👍